include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -367,12 +367,12 @@ namespace LinearAlgebra
        * This function is mainly for internal consistency checks and should
        * seldom be used when not in debug mode since it uses quite some time.
        */
-      virtual bool all_zero () const override;
+      bool all_zero () const override;
```

Comment:
I would like to see the `virtual` annotation be retained. Not everyone is aware about every language detail (i.e. `override` being a valid qualifier only for a virtual function) and a bit of redundancy in that regard doesn't hurt :-)